CHANGELOG.md

```diff
@@ -1,3 +1,114 @@
+# v1.13.0 (2022-03-??)
+
+## OS Changes
+
+* Set NVMe IO request timeouts for EBS according to AWS recommendations ([#2820])
+* Add `ethtool` to release ([#2829])
+* Add `NoOpMigration` for migrations and migration-helpers
+  for when no data migration is necessary _(Also cherry-picked to v1.12.0)_ ([#2749])
+* Add info logging to migrator in order to document onging migrations ([#2751])
+* Rename `ncurses` to `libncurses` ([#2769])
+* Add `CAP_SYS_MODULE` and `CAP_CHROOT` to bootstrap containers ([#2772])
+* Upgrade `tokio` to latest LTS in Bottlerocket sources and "latest" in tools ([#2736])
+* Add support for cgroup v2 to NVIDIA variants ([#2802])
+* Disable IA and SafeSetID LSM for kernel-5.15 ([#2789])
+* Update kernel-5.10 to 5.10.165 and kernel-5.15 to 5.15.90 ([#2795])
+* Allow `=` in bootconfig values ([#2806])
+* Upgrade `go-restful` code dependency used by containerd ([#2813])
```

Comment:
This one might be worth stating why a containerd dependency is being updated separately from containerd, to address the security issue that is not a security issue.